auth_totp/models/res_users.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from datetime import datetime, timedelta
```

Comment:
As far as I can tell, the order is correct, so is it just the lack of whitespace between groups that's a problem? Since there are only 6 lines, I felt that having them in one block would be fine.